dianna/methods/lime.py

```diff
@@ -276,3 +282,27 @@ def moveaxis_function(data):
         if self.preprocess_function is None:
             return moveaxis_function
         return lambda data: self.preprocess_function(moveaxis_function(data))
+
+    @staticmethod
```

Comment:
```suggestion
```
A hidden function would be enough since we encourage users to call dianna API rather than call the explainer directly, so not necessary for a static method.